### PR TITLE
Version the documentation

### PR DESCRIPTION
**Pull request by cdrage**
**What type of PR is this:**

/kind documentation

**What does this PR do / why we need it:**

This versions the documentation in order to display both 2.5.0 and 3.0.0
alpha1

In this PR we:

* Update the package dependencies, as there was a small bug with
  docusaurus displaying versioning
* Moves all documentation for 3.0.0-alpha1 into it's own folder
* Makes the old 2.5.0 documentation as the most current docs

**Which issue(s) this PR fixes:**

Fixes https://github.com/redhat-developer/odo/issues/5377

**PR acceptance criteria:**

- [X] Unit test

- [X] Integration test

- [X] Documentation

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

File: docs/website/docs/command-reference/create.md [hunk 1]
```diff
@@ -53,8 +53,6 @@ odo create nodejs --starter nodejs-starter
 
 This will download the example template corresponding to the chosen component type (in the example above, `nodejs`) in your current directory (or the path provided with the `--context` flag).
 
-If a starter project has its own devfile, then this devfile will be preserved.
```
Comment (feloy) [inline docs/website/docs/command-reference/create.md:56]: All these modified files in `docs/website/docs/` comes unmodified from the v2 branch, right?

**Comment by dharmit** [comment]
Love this little thing: 
image


/approve

**Comment by kadel** [comment]
> * Moves all documentation for 3.0.0-alpha1 into it's own folder

@cdrage how is this going to work when someone makes changes in main branch? Where it will show up?

**Pull request merged**